docs/attributes-registry/user-agent.md

```diff
@@ -8,5 +8,12 @@
 <!-- semconv registry.user_agent(omit_requirement_level) -->
 | Attribute  | Type | Description  | Examples  |
 |---|---|---|---|
+| `user_agent.device.name` | string | Name of the device extracted from original | `iPhone` |
```

Comment:
maybe `user_agent.platform.name` (based on https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-CH-UA-Platform)?

Reply:
@MSNev is it `browser.brands` in this case for iPhone?

Reply:
It will be for available for any browser environment that supports it (based on the instrumentation populating it) -- so based on the current "known" state of the browser API https://caniuse.com/?search=brands, if the user is using Safari then No it wont be available.

Reply:
That "doesn't" mean though that some instrumentation could construct and populate this fields as it's defined by OpenTelemetry. So it could be present.

Reply:
I think then these things can work independently. One is taking data out of browser API, another extracts it from `user_agent.original` for easy processing later and they could have different values. So then I think it makes sense to have presented in this PR fields `name` and `version` to be added.